Directory.Packages.props

```diff
@@ -195,11 +195,11 @@
     <PackageVersion Include="Microsoft.ApplicationInsights.WorkerService" Version="2.22.0" />
 
     <!-- Pinned versions for Component Governance - Remove when root dependencies are updated -->
-    <PackageVersion Include="System.Text.Json" Version="8.0.4" />
     <PackageVersion Include="Azure.Core" Version="1.44.0" />
     <PackageVersion Include="Azure.Identity" Version="1.12.1" />
     <!-- https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3313 -->
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageVersion Include="System.Formats.Asn1" Version="8.0.1" />
+    <PackageVersion Include="System.Formats.Asn1" Version="$(SystemFormatsAsn1PackageVersion)" />
```

Comment:
Similar comment to your PR in main, but should we avoid using properties and instead spell out the versions here? This is the reasoning behind my question: Next time that we are bumping dependencies, if the version bump that I see is under this section of Pinned for CG, then I'd likely undo the change, as the long term goal is to remove these as opposed to continue bumping them (once the direct dependency updates the transitive dependency version). If the update is in Versions.props in a property, I'm unlikely to make that link and decide not to update, causing is to always be pinning this transitive dependency.

Reply:
I honestly don't know when we are supposed to be doing logic in Directory.Packages.props and when we are supposed to be doing it in Versions.props. I was just following how we did the split with EF based on TFMs. See https://github.com/dotnet/aspire/pull/6190#discussion_r1792522231